### DISCUSSION TITLE
earning stars

### DISCUSSION THREAD

**Discussion by giamimino** [community/community#141484] [New to GitHub]
i am new on **github** how to earn **stars**?

**Comment by mctrinh** [comment]
I think the good way to earn stars is to focus on creating valuable projects, writing clear documentation, and engaging with the community.
Collaborate, share updates, and promote your work to get noticed!

**Comment by lin-ycv** [comment]
stars are give to a repo by other users, it's like giving a like on social media.

**Comment by MahdiRahmatii** [comment]
You can ask your friends and support each other. Also, get feedback from your friends to improve the project

**Comment by Himani0002** [comment]
I am a full stack development student and BCA.

Tell me about any internship or anything you might know.

please

**Reply by MahdiRahmatii** [reply]
Hello. This depends a lot on where you live. But you can usually get good results on LinkedIn. On the other hand, follow related Telegram groups and use job search websites.

**Reply by MahdiRahmatii** [reply]
And most importantly, write a good resume and introduce yourself accurately

**Reply by MahdiRahmatii** [reply]
Internship benefits: 

Skill Development: Interns gain industry-specific skills and improve general competencies like communication, teamwork, and problem-solving.

Networking: Interns build professional connections that can lead to future job opportunities or mentorship.

Resume Enhancement: Internships provide credible experience to add to a resume, making candidates more attractive to employers.

**Comment by UtkarshK95** [comment]
To earn stars on GitHub, start by creating projects that others might find useful or interesting. Write a clear README with examples to make your project easy to understand. Share your work on Twitter, Reddit, or dev communities to get more eyes on it. Contribute to other projects, too—it builds connections, and people might check out your profile. Finally, respond to any feedback, showing that you’re active and open to ideas. Keep it up, and the stars will come!
